roxctl/netpol/connectivity/diff/README.md

```diff
@@ -0,0 +1,115 @@
+# Static Network Policy Connectivity Diff
```

Comment:
It's great to see that the bar is being raised and we have a lovely doc. I wonder whether it is discoverable enough, maybe link it in the help output?

Reply:
Thanks @rukletsov ! I think, Boaz is working on a proper documentation that will be targeted for a future release >4.2. For the tech preview phase of that, we agreed to stay with a readme in the folder where a feat is implemented.

We are happy to make it easier to find if you have any particular places in mind where we could link that.